web/sites/default/config/views.view.council_documents.yml

```diff
@@ -723,19 +723,6 @@ display:
       sorts: {  }
       arguments: {  }
       filters:
-        status:
```

Comment:
I think this filter may have been mistakenly removed, we want a Published = True filter on the default view.